docs/use-cases.md

```diff
@@ -1,53 +1,17 @@
-# Use Cases
+# Why Ark?
 
-This doc provides sample Ark commands for the following common scenarios:
-* [Disaster recovery][0]
-* [Cluster migration][1]
+Kubernetes workloads are ephemeral. You describe your cluster's state, and Kubernetes makes it so. So why do you still need to worry about backups and restores? Ark addresses these use cases:
 
-## Disaster recovery
+* Disaster recovery for your persistent volumes
 
-*Using Schedules and Restore-Only Mode*
+* If you don't start with your Kubernetes manifests in source control, you can run Ark to download the entire cluster and then put the manifests in source control.
 
-If you periodically back up your cluster's resources, you are able to return to a previous state in case of some unexpected mishap, such as a service outage. Doing so with Heptio Ark looks like the following:
+* Copy your environment by cloning a cluster, or a portion of a cluster, to another cluster or to the same cluster. You can use the copy for scale testing, reproducing bugs, and more.
 
-1. After you first run the Ark server on your cluster, set up a daily backup (replacing `<SCHEDULE NAME>` in the command as desired):
+  * If you specify CPU and memory requests and limits for a pod, you can test these settings with a clone.
 
-    ```
-    ark schedule create <SCHEDULE NAME> --schedule "0 7 * * *"
-    ```
-    This creates a Backup object with the name `<SCHEDULE NAME>-<TIMESTAMP>`.
+* Support for backing up any directory in any container's file system is planned for a future release. This feature will let you migrate your data across cloud providers more easily.
```

Comment:
Should we include this here, now?

Reply:
Yeah, wasn't sure. Tossed it in for discussion. Default Official Docs position: no, we shouldn't.

Reply:
Ok yeah I'd remove